index.html

```diff
@@ -27,18 +27,16 @@ <h1><img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc
                               </ul>
                             </nav>
                           </header>
-                         
+                       
                           
                 </div>
                 <br>
                 <br>
-                </div>
-                </header>
                 <div class="handle-Trombones">

```

Comment:
Please give a proper name to your classes. Classes and ids should be in lowercase. Thanks! 😄

Reply:
I think your comment is not clear enough. Giving clear feedback helps with effective communication, instead of asking to make a change only also add "why" the changes are necessary. See this article on [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) for more details

Reply:
nicely done 👍